context/app/api/vitessce_confs/test_confs.py

```diff
@@ -0,0 +1,49 @@
+import json
+import glob
+from pathlib import Path
+import re
+
+from .base_confs import ImagePyramidViewConf
+from .assay_confs import (
+    SeqFISHViewConf,
+    CytokitSPRMConf,
+    RNASeqConf,
+    ATACSeqConf,
+    IMSConf,
+)
+
+MOCK_NEXUS_TOKEN = "nexus_token"
+
+FIXTURES_INPUT_DIR = "fixtures/input_entity"
+FIXTURES_EXPECTED_OUTPUT_DIR = "fixtures/output_expected"
+
+AssayConfClasses = {
+    "codex": CytokitSPRMConf,
+    "rna": RNASeqConf,
+    "atac": ATACSeqConf,
+    "ims": IMSConf,
+    "image_pyramid": ImagePyramidViewConf,
+    "seqfish": SeqFISHViewConf,
+}
+
+
+def test_assays():
+    parent_dir = Path(__file__).parent
+    for entity_file in glob.glob(
+        str(parent_dir / f"{FIXTURES_INPUT_DIR}/*_entity.json")
+    ):
+        assay = re.search(
+            str(parent_dir / f"{FIXTURES_INPUT_DIR}/(.*)_entity.json"), entity_file,
```

Comment:
Unclear on whether `FIXTURES_INPUT_DIR` is a regex or not: 
- If it's not, it should be escaped.
- If it is, it should be something like `FIXTURES_INPUT_DIR_RE`

Reply:
Done